src/explore-education-statistics-admin/src/pages/release/datablocks/__tests__/ReleaseDataBlockEditPage.test.tsx

```diff
@@ -119,6 +119,7 @@ describe('ReleaseDataBlockEditPage', () => {
             dataSets: [],
           },
         },
+        boundaryLevel: undefined,
```

Comment:
Is it necessary to include this is `boundaryLevel` is optional?

Reply:
Yep, I don't understand why, maybe some TS quirk. I tried with null/undefined but it still insisted on it being declared.
![image](https://github.com/user-attachments/assets/65d500a5-90b0-4b02-b078-f24c951b36f0)